.yarnrc

```diff
@@ -1 +1 @@
-save-prefix false
+save-prefix ""
```

Comment:
?

Reply:
Weird addition that whenever you add a new npm package, it prefixes the package version with 'false' in front of it.

Link to issue - https://github.com/yarnpkg/yarn/issues/4104